ui/plugin-system/src/components/Variables/VariableEditorForm/variable-editor-form-model.ts

```diff
@@ -13,18 +13,44 @@
 
 import { TextVariableDefinition, VariableDefinition } from '@perses-dev/core';
 
-export function getInitialState(initialVariableDefinition: VariableDefinition) {
-  const textVariableFields = {
+interface textVariableFields {
```

Comment:
isn't it possible to reuse the variable model interfaces defined in the core pkg here?

Reply:
Yup